cluster-autoscaler/metrics/metrics.go

```diff
@@ -39,10 +39,11 @@ type FunctionLabel string
 type NodeGroupType string
 
 const (
-	caNamespace   = "cluster_autoscaler"
-	readyLabel    = "ready"
-	unreadyLabel  = "unready"
-	startingLabel = "notStarted"
+	caNamespace           = "cluster_autoscaler"
+	readyLabel            = "ready"
+	unreadyLabel          = "unready"
+	startingLabel         = "notStarted"
+	longUnregisteredLabel = "long_unregistered"
```

Comment:
s/long_unregistered/longUnregistered. We use camelCase in metric labels.

Reply:
Done